packages/kernel/src/kernel.ts

```diff
@@ -164,18 +167,45 @@ export class StliteKernel {
     this.onModuleAutoLoad = options.onModuleAutoLoad;
 
     if (options.worker) {
-      this._worker = options.worker;
+      this._workerState = {
+        worker: options.worker,
+        ports: new Set(),
```

Comment:
Why is this managed as a `Set`, while it seems that the port instance we will have is just one?
LMK if it's wrong.